hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -67,7 +71,8 @@ public TpcServerBootstrap(NodeEngineImpl nodeEngine) {
         this.nodeEngine = nodeEngine;
         this.logger = nodeEngine.getLogger(TpcServerBootstrap.class);
         this.ss = (InternalSerializationService) nodeEngine.getSerializationService();
-        this.enabled = Boolean.parseBoolean(getProperty("hazelcast.tpc.enabled", "false"));
+        this.enabled = Boolean.parseBoolean(getProperty("hazelcast.tpc.enabled", "false"))
```

Comment:
The logic for checking the System properties should be part of the tpcEngineConfig.

Reply:
I can move it. But how about removing this property with eventloop count? In the future when xml/yaml related stuff is implemented, there will be also this: https://docs.hazelcast.com/hazelcast/5.2/configuration/configuring-with-system-properties.